packages/apps/src/directory/DirectoryTemplate.tsx

```diff
@@ -69,11 +69,19 @@ const DirectoryTemplate: React.FC<DirectoryTemplateProps> = (props) => {
   const [showMap, setShowMap] = useState(true)
   const [expandMap, setExpandMap] = useState(false)
 
+  const isListMode = variant === PaginatedQueryViewVariantEnum.List
+
   // State: Bottom drawer
   const directoryListingsJsx = (
     <PaginatedListings
-      gridItemProps={gridItemProps}
-      gridProps={gridProps}
+      gridItemProps={
+        isListMode
```

Comment:
Are we able to manage these scenarios in the switch case instead? Want to keep the view clean